test/utilities.test.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const tap = require('tap')
+const utilities = require('../lib/components/utilities')
+
+tap.test('Utility functions', t => {
+  t.beforeEach((done, t) => {
+    done()
+  })
+
+  t.afterEach((done, t) => {
+    done()
+  })
```

Comment:
These are unnecessary.

Reply:
Yes, in the current state indeed. I left them in as template for someone who might add tests to this module in the future. But I can remove them for now of course.

Reply:
Please. I'd rather not add code for _potential_ future cases. The code should reflect the current state.

Reply:
The unnecessary code has been removed.